extensions/stackdriver/stackdriver.cc

```diff
@@ -62,6 +63,20 @@ constexpr int kDefaultTickerMilliseconds = 10000;  // 10s
 
 namespace {
 
+const RE2 authz_name_match("ns\\[(.*)\\]-policy\\[(.*)\\]-rule\\[(.*)\\]");
+constexpr char kRbacAccessAllowed[] = "AuthzAllowed";
+constexpr char kRbacAccessDenied[] = "AuthzDenied";
+constexpr char kRBACHttpFilterName[] = "envoy.filters.http.rbac";
+constexpr char kRBACNetworkFilterName[] = "envoy.filters.network.rbac";
+constexpr char kDryRunDenyShadowEngineResult[] =
```

Comment:
These keys are defined in control plane?

Reply:
the prefix `istio_dry_run_deny_` is defined in istiod and the remaining part `shadow_engine_result` is defied in Envoy RBAC filter.